app/validators/questionnaire_validator.py

```diff
@@ -104,3 +104,22 @@ def validate_smart_quotes(self):
                         error_messages.DUMB_QUOTES_FOUND,
                         pointer=translatable_item.pointer,
                     )
+
+    def validate_white_spaces(self):
+        schema_object = SurveySchema(self.schema_element)
+
+        for translatable_item in schema_object.translatable_items:
+            schema_text = translatable_item.value
+            values_to_check = [schema_text]
+
+            if isinstance(schema_text, dict):
+                values_to_check = schema_text.values()
+
+            for text in values_to_check:
+                if text and (
+                    text.startswith(" ") or text.endswith(" ") or "  " in text
+                ):
```

Comment:
Is it not possible for text to be a non string at this point?. This can be `schema_text.values()` which could include non strings?.